### PR TITLE
Document insert overwrite limitation

### PR DESCRIPTION
**Pull request by raunaqmorarka**

### CODE REVIEW

File: presto-docs/src/main/sphinx/migration/from-hive.rst [hunk 1]
```diff
@@ -146,3 +146,6 @@ query::
 
 The resulting behavior is equivalent to using `INSERT OVERWRITE
 <https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DML>`_ in Hive.
+
+Overwrite behavior for existing partitions is currently not supported by Presto
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/migration/from-hive.rst:150]: Not sure about the terms "overwrite behavior" ... maybe instead use 

Overwrite operations of existing partitions ...

Comment (mosabua) [reply presto-docs/src/main/sphinx/migration/from-hive.rst:150]: also remove the redundant "currently"

**Review by mosabua — APPROVED** [review]
Looks good to me now. Short and simple.

**Review by sopel39 — APPROVED** [review]

**Comment by sopel39** [comment]
merged, thanks!

**Pull request merged**